pkg/network/state_test.go

```diff
@@ -320,6 +322,9 @@ func TestLastStatsForClosedConnection(t *testing.T) {
 	conn2.MonotonicRecvBytes += dRecv
 	conn2.MonotonicRetransmits += dRetransmits
 
+	// Let's register our client first
```

Comment:
You can remove these comments `// Let's register our client first`. They don't provide anything above what the code already says.